src/consensus/consensus.rs

```diff
@@ -22,6 +21,13 @@ pub enum SystemMessage {
     DecidedValueForReadNode(proto::DecidedValue),
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ValidatorSetConfig {
+    pub effective_at: u64,
+    pub validator_public_keys: Vec<String>,
+    pub shard_ids: Vec<u32>,
```

Comment:
Could you update `setup_local_testnet` with the new format?

Reply:
Done!